### PR TITLE
News 2021 11

### PR DESCRIPTION
**Pull request by maelle**
TODOS

- [x] Rebase to get the cffr posts.

### CODE REVIEW

File: content/blog/2021-11-30-ropensci-news-digest-november-2021/index.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+<!-- Before sending DELETE THE INDEX_CACHE and re-knit! -->
+
+Dear rOpenSci friends, it's time for our monthly news roundup!
+<!-- blabla -->
+You can read this post [on our blog](/blog/2021/11/30/ropensci-news-digest-november-2021).
+Now let's dive into the activity at and around rOpenSci!
+
+## rOpenSci HQ
+
+### Community call. @ropensci-review-bot help!
```
Comment (steffilazerte) [inline content/blog/2021-11-30-ropensci-news-digest-november-2021/index.md:27]: I was a bit confused by this heading, perhaps a colon will connect the two a bit better?
```suggestion
### Community call: @ropensci-review-bot help!
```

File: content/blog/2021-11-30-ropensci-news-digest-november-2021/index.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+
+Speakers [Arfon Smith](https://www.arfon.org/), [Karthik Ram](/author/karthik-ram), [Maëlle Salmon](/author/maëlle-salmon) and [Noam Ross](/author/noam-ross) will discuss how rOpenSci has worked with [The Journal of Open Source Software (JOSS)](https://joss.theoj.org/) to extend their approach of [chatops-driven publishing](https://www.arfon.org/chatops-driven-publishing) into a new GitHub chat-bot that manages our editorial process: assigning tasks, tagging issues, running tests on software submissions, and returning reports to reviewers and editors, all from the comfort of a GitHub issue comment. Chat-ops automation has been critical to JOSS’s success, and shows huge promise for automating even more tedious tasks like frequent software checks and compliance with best practices. We will show how our new review bot improves the author and editor experience, **demonstrate how other teams can use and customize the bot**, and discuss how this approach can be extended to tackle other automation, code review, and reproducibility challenges.
+
+Lots of time for Q & A during the session.
```
Comment (steffilazerte) [inline content/blog/2021-11-30-ropensci-news-digest-november-2021/index.md:33]: During, or at the end?
```suggestion
Come join the conversation, we'll have lots of time for Q & A during the session.
```

File: content/blog/2021-11-30-ropensci-news-digest-november-2021/index.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+
+### rOpenSci help wanted issues
+
+Help Wanted issues have a home: our website's new [help-wanted page](/help-wanted/).
```
Comment (steffilazerte) [inline content/blog/2021-11-30-ropensci-news-digest-november-2021/index.md:37]: I'm excited :wink:
```suggestion
Help Wanted issues have a home: our website's new [help-wanted page](/help-wanted/)!
```

File: content/blog/2021-11-30-ropensci-news-digest-november-2021/index.md [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+
+This page lists ALL recent(ish) issues from rOpenSci R packages labelled "help wanted". We'll still [tweet new issues](https://twitter.com/search?q=from%3Aropensci%20%5Bhelp%20wanted%5D&f=live)!
```
Comment (steffilazerte) [inline content/blog/2021-11-30-ropensci-news-digest-november-2021/index.md:39]:
```suggestion
This page lists ALL recent(ish) issues from rOpenSci R packages labelled "help wanted". We'll still [tweet new issues](https://twitter.com/search?q=from%3Aropensci%20%5Bhelp%20wanted%5D&f=live), but this is a great place to see what's out there that needs your help.
```

File: content/blog/2021-11-30-ropensci-news-digest-november-2021/index.md [hunk 5]
```diff
@@ -0,0 +1,239 @@
...
+
+- Sort e.g, good first issue;
+- Search e.g., specific packages.
```
Comment (steffilazerte) [inline content/blog/2021-11-30-ropensci-news-digest-november-2021/index.md:42]: In the tweet I was pretty brief due to character constraints, but I think we can expand a bit more here. 
```suggestion

On the page you can easily find issues relevant to you by:  

- Sorting (e.g, find all the issues labelled "good first issue");
- Searching (e.g., look for issues in your favourite packages)
```

File: content/blog/2021-11-30-ropensci-news-digest-november-2021/index.md [hunk 6]
```diff
@@ -0,0 +1,239 @@
...
+
+### Next co-working sessions
+
+Join us for social coworking & office hours monthly on 1st Tuesdays! Hosted by Steffie LaZerte, Nick Tierney, and Stefanie Butland. Everyone welcome. No RSVP needed. Consult our [Events](/events) page to find your local time and how to join.
+
+Our next sessions are
+
+* [Monday, 06 December 2021 17:00 PST](/events/coworking-2021-12/);
+* [Tuesday, 04 January 2022 09:00 PST](/events/coworking-2022-01/).
```
Comment (steffilazerte) [inline content/blog/2021-11-30-ropensci-news-digest-november-2021/index.md:51]: Feel free to omit if you think this is too redundant!
```suggestion
* [Monday, 06 December 2021 17:00 PST](/events/coworking-2021-12/) (9 AM Australian Western);
* [Tuesday, 04 January 2022 09:00 PST](/events/coworking-2022-01/) (9 AM North American Pacific).
```

File: content/blog/2021-11-30-ropensci-news-digest-november-2021/index.md [hunk 7]
```diff
@@ -0,0 +1,239 @@
...
+
+## Call for maintainers
+
+<!--IF CALL
+* [our guidance on _Changing package maintainers_](https://devguide.ropensci.org/changing-maintainers.html)
+* [our _Package Curation Policy_](https://devguide.ropensci.org/curationpolicy.html)
+
+IF NO CALL
+There's no open call for new maintainers at this point but you can refer to our [contributing guide](https://contributing.ropensci.org/) for finding ways to get involved!
+As the maintainer of an rOpenSci package, feel free to contact us on Slack or email `info@ropensci.org` to get your call for maintainer featured in the next newsletter. -->
```
Comment (steffilazerte) [inline content/blog/2021-11-30-ropensci-news-digest-november-2021/index.md:164]: I'm sure you're on top of it, but this comment is a reminder to update this section :)

File: content/blog/2021-11-30-ropensci-news-digest-november-2021/index.md [hunk 8]
```diff
@@ -0,0 +1,239 @@
...
+
+## Package development corner
```
Comment (steffilazerte) [inline content/blog/2021-11-30-ropensci-news-digest-november-2021/index.md:166]: This section always amazing me, thanks so much!

File: content/blog/2021-11-30-ropensci-news-digest-november-2021/index.Rmd [hunk 9]
```diff
@@ -0,0 +1,619 @@
...
+      string, sprintf(
+        " It is available on [Bioconductor](https://bioconductor.org/packages/%s/). ",
+        package$package
+      )
+    )
+  }
+  if (nzchar(package$onboarding)) {
+    string <- paste0(string, sprintf("It has been [reviewed](%s)", package$onboarding))
+    if (length(rev) > 0) {
+      string <- paste0(string, sprintf(" by %s.", toString(rev)))
```
Comment (steffilazerte) [inline content/blog/2021-11-30-ropensci-news-digest-november-2021/index.Rmd:208]: I'm not sure you want to add more dependencies, but if you use glue, you can have:
```
glue_collapse(c("Steffi", "Maëlle", "Stef"), sep = ", ", last = ", and ")
#> Steffi, Maëlle, and Stef
```

File: content/blog/2021-11-30-ropensci-news-digest-november-2021/index.Rmd [hunk 10]
```diff
@@ -0,0 +1,619 @@
...
+Help Wanted issues have a home: our website's new [help-wanted page](/help-wanted/).
+
+This page lists ALL recent(ish) issues from rOpenSci R packages labelled "help wanted". We'll still [tweet new issues](https://twitter.com/search?q=from%3Aropensci%20%5Bhelp%20wanted%5D&f=live)!
+
+- Sort e.g, good first issue;
+- Search e.g., specific packages.
+
+### Next co-working sessions
+
+Join us for social coworking & office hours monthly on 1st Tuesdays! Hosted by Steffie LaZerte, Nick Tierney, and Stefanie Butland. Everyone welcome. No RSVP needed. Consult our [Events](/events) page to find your local time and how to join.
```
Comment (steffilazerte) [inline content/blog/2021-11-30-ropensci-news-digest-november-2021/index.Rmd:61]: I didn't even notice this until I ran a spell check 🤦🏼‍♀️ 
```suggestion
Join us for social coworking & office hours monthly on 1st Tuesdays! Hosted by Steffi LaZerte, Nick Tierney, and Stefanie Butland. Everyone welcome. No RSVP needed. Consult our [Events](/events) page to find your local time and how to join.
```

**Review by steffilazerte — COMMENTED** [review]
Looks good! Love the dev corner, as usual!

* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md

File: content/blog/2021-11-30-ropensci-news-digest-november-2021/index.md (hunk 2 above)
Comment (stefaniebutland) [reply content/blog/2021-11-30-ropensci-news-digest-november-2021/index.md:33]: I boorishly stepped in and edited Steffi's suggested change.

**Pull request merged**